app/assets/stylesheets/_buttons.scss

```diff
@@ -70,9 +70,14 @@ a .button,
 .orange_submit_button { padding: 5px; }
 .translation_button { @extend .play_video_button; }
 
+// Browser specific styles ( Firefox IE 11 and Safari ) to hide the default videojs BigPlayButton
+// User expects to view video without any obsufication and the default button was obsuring the video
 .video-js.vjs-big-play-button {
   display: none;
 }
+
+// Browser specific style for Chrome to hide the default videojs BigPlayButton
+// User expects to view video without any obsufication and the default button was obsuring the video
 video::-webkit-media-controls-overlay-play-button {
   display: none;
```

Comment:
🐘 You probably already looked into this, but would this also benefit from `-webkit-appearance: none;`?

Reply:
I'm sure it can't hurt to add it in right?